agora-scripts/agora-scripts.json

```diff
@@ -0,0 +1 @@
27563746f720022233330040020032333300600200120011480005200022333300423371e6eb8c07c00400c004888c008dd598128018912800911999801919b8f375c603c00200600244460046eb4c09000c4894004c0508894ccc060004489400454ccc064c008c07c004488c008c08400c4cc00c008c0780052f5c091011c8fd3f732c764550a66a42f31e5936b6dc9f85eabb5c08c774cebd3cd0048810022333003223330067000040020040024446600a6600c4646666022002006466ec0008dd3198031bab301f0033756002200460320020046600e4601466016466ebcc068004c06800800c004888cc010cc0148c8cccc04000400c8cdd80011ba833006375a603c0066eb40044008c060004008cc0188c024cc0288cdd7980c800980c80100180091198079129998090008801899803980c8009801180c0008011180691299980800088050998029801980b8009801180b000918061129998078008804899299980898020008998028009801980b00109801980b001180b000aba0233300c0014a09448c024894ccc0300045280a99802180198098008980118090009198050008010a514bd6f7b6301119998019ba9002001233300522337006eb4c04c008005200037560022900011199980291299980400089128008a99980499baf300d300f0010041223002301100313002300e0010012223002301200312250012230042253330070011003133004300e0013002300d00148811c55d252e7c81d25e68ed198a26f07832f838c4d1f078e7d28bd18c209002323002233002002001230022330020020015573eae688c014c0080048c010c0100055ce2ab9d5744ae848c008dd5000aab9e1"},"treasuryValidatorInfo":{"hash":"39ad85486d8624a49f080636324f80938febfb91552709c70a3ac00e","script":"590338010000323232323232323232323232323222232323232533300e3370e90000010992999807a99806992999808000880089980924813373696e676c65417574686f72697479546f6b656e4275726e65643a204d757374206275726e2065786163746c79203120474154000013370e6466660146ea40440048ccc88c03c894ccc048004400c4cc010c064004c008c06800488cdc01bad301a00200148000dd58008a40006eacc04cc8c054c054c054004c054004cdc0a400090010992999808000880089980924814673696e676c65417574686f72697479546f6b656e4275726e65643a20416c6c2047415420746f6b656e73206d7573742062652076616c69642061742074686520696e70757473000013300a23232333300c37520266eacc058c05c0048c8c8c94ccc058cdc3a4004004264a66602e00220022660329213e617574686f72697479546f6b656e7356616c6964496e3a2047415420546f6b656e4e616d6520646f65736e2774206d617463682053637269707448617368000013301123375e60340026ea4dd7180d8011bab0041325333017001100113301949137617574686f72697479546f6b656e7356616c6964496e3a2047415420696e636f72726563746c79206c69766573206174205075624b6579000014a06038004602e0026ea8c05cc064c05c008528980b80099180a980b000980b0009bac30133015001149854cc04524128412073696e676c6520617574686f7269747920746f6b656e20686173206265656e206275726e65640016301200415330104901465061747465726e206d61746368206661696c75726520696e2027646f2720626c6f636b2061742061676f72612f41676f72612f54726561737572792e68733a38353a332d313200163014002300f0013754601e6020002602000244666600844a66600e002244a0022a66601466ebcc034c038004010488c008c04000c4c008c03c004004888c008c04800c48940048c008894ccc0140045288a9980318019806000898011806800919180111980100100091801119801001000aab9f2233300400200100314a0ae692211c6aa44a4ea34725d9bcf9b522babe6b6b94ae55eddac23b9ccaa94427005738aae755d0aba2230023754002aae781"}}
```

Comment:
I assume this is the result of calling the export command. Should it be cached in the repo? It will require manual intervention every time there's a change to Agora's scripts, right?

Reply:
Yeah, that's true. I think it's okay, it's a temporary solution until something nicer comes along. The same issue exists for the purescript-bridge executable too.